Add balance command

jrchatruc opened the pull request:
# What :computer: 
* Add `npx zksync-cli wallet balance` command to check ETH balance of the specified address

# Why :hand:
* Seeing ETH balance might be useful during the development

# Evidence :camera:
<img width="452" alt="image" src="https://github.com/matter-labs/zksync-cli/assets/47187316/6beaca52-bee0-4cd7-9f6a-7c894d9c251b">

JackHamer09 commented on src/commands/wallet/command.ts:
```diff
@@ -0,0 +1,3 @@
+import Program from "../../program.js";
+
+export default Program.command("wallet").description("Manage wallet related features for L2 and L1");
```
For now there is no L1 features.
So we should either not mention L1 and just say `Manage zkSync wallet related features`
or
Add possibility to choose L1 network in the list. For example:
```
❯ zkSync Era Testnet 
  zkSync Era Mainnet 
  Local Dockerized node
  ---
  Ethereum Mainnet
  Ethereum Goerli Testnet
```

JackHamer09 commented on src/commands/wallet/balance.ts:
```diff
@@ -0,0 +1,74 @@
+        const Provider = getL2Provider(options.l2RpcUrl ?? toChain!.rpcUrl);
+        const balance = await Provider.getBalance(options.account ?? "Unknown account");
+
+        Logger.info(`\n${bigNumberToDecimal(balance)} ETH`);
```
Let's say `${selectedChain.name} Balance: XX ETH`

JackHamer09 commented on src/commands/wallet/balance.ts:
```diff
@@ -0,0 +1,74 @@
+            ...answers,
+        };
+
+        const toChain = l2Chains.find((e) => e.network === options.chain);
```
Let's call it `selectedChain`

JackHamer09 commented on src/commands/wallet/balance.ts:
```diff
@@ -0,0 +1,74 @@
+        };
+
+        const toChain = l2Chains.find((e) => e.network === options.chain);
+        const Provider = getL2Provider(options.l2RpcUrl ?? toChain!.rpcUrl);
```
Let's not create uppercase variables

JackHamer09 commented on src/commands/wallet/balance.ts:
```diff
@@ -0,0 +1,74 @@
+
+import Program from "./command.js";
+import {
+    accountOption,
```
Try to properly setup prettier so the code is properly styled

JackHamer09 commented on src/common/options.ts:
```diff
@@ -26,6 +27,12 @@ export type DefaultTransactionOptions = DefaultOptions & {
   l2RpcUrl?: string;
   privateKey: string;
 };
+export type BalanceOptions = DefaultOptions & {
```
I don't think we should keep `BalanceOptions` in this file. Let's move it to balance.ts where it is used.

JackHamer09 reviewed (changes requested):
Should we also ask for which token to get balance to? ETH or ERC20
And then if ERC20 is selected we also ask for address of the token.
Would be useful to have something like that :)
Not required for this PR tho..

jrchatruc commented:
> Should we also ask for which token to get balance to? ETH or ERC20 And then if ERC20 is selected we also ask for address of the token. Would be useful to have something like that :) Not required for this PR tho..

That would be very useful indeed, we'll add it on a subsquent PR!

jrchatruc commented:
Addressed all the commends above here [aef66ef](https://github.com/matter-labs/zksync-cli/pull/79/commits/aef66ef45749923c48763bc356c0c36a87e32b41) and here [2e88921](https://github.com/matter-labs/zksync-cli/pull/79/commits/2e8892186e7e3538e4433467d4dae6970b90cf63). In regards to this

```
For now there is no L1 features.
So we should either not mention L1 and just say Manage zkSync wallet related features
or
Add possibility to choose L1 network in the list.
```

For now I just changed the description to not mention L1, we might add L1 features in the future.
